editor/src/core/shared/element-template.ts

```diff
@@ -1114,6 +1114,12 @@ export function isJSXElementLike(element: JSXElementChild): element is JSXElemen
   return isJSXElement(element) || isJSXFragment(element)
 }
 
+export type JSXNonDomElement = JSXFragment | JSXConditionalExpression
+
+export function isNonDomElement(element: JSXElementChild): element is JSXNonDomElement {
```

Comment:
as part of the original fix I added this, but then we ended up not using this. The same thing happened two weeks ago when Berci's PR added a function similar to this that we ended up not using and removing. 

I now wonder if this should be considered a code smell if in the future we end up needing this helper function.